bb/ramfs.go

```diff
@@ -115,23 +117,57 @@ func ramfs() {
 
 	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
 
-	if *extraPaths != "" {
-		extras := strings.Split(*extraPaths, " ")
-		for _, x := range extras {
-			p := strings.Split(x, ":")
-			if len(p) != 2 {
-				p = append([]string{"/"}, p...)
+	if *extraCmds != "" {
+		copyc := strings.Fields(*extraCmds)
+		//check if the path is a file or directory
+		for _, eachPath := range copyc {
+			//must have absolute filepath for os.Stat
+			statval, err := os.Stat(eachPath)
+			if err != nil {
+				log.Fatalf("%v\n", err)
+			}
+			if statval.IsDir() {
+				//treat the file as if it were in extraPaths
+				p := strings.Split(eachPath, ":")
+				if len(p) != 2 {
+					p = append([]string{"/"}, p...)
+				}
+				paths[p[0]] = append(paths[p[0]], p[1])
+			} else {
+				if err != nil {
+					log.Fatalf("%v\n", err)
+				}
+				copyCommands(w, eachPath)
 			}
-			paths[p[0]] = append(paths[p[0]], p[1])
 		}
 	}
 
-	if *extraCmds != "" {
-		copyCommands(w, strings.Split(*extraCmds, " "))
-	}
+	/*
```

Comment:
Don't keep dead code around. Just remove it. We can use git history to see what used to be there if we need to.